### ISSUE TITLE
Test Error

### ISSUE THREAD

**Issue by OK1SLM** [projecthorus/horusdemodlib#239]
The following tests FAILED:
	  3 - test_horus_rtty_7n1 (Failed)
	  4 - test_horus_rtty_7n2 (Failed)
	  5 - test_horus_rtty_8n2 (Failed)

**Comment by darksidelemm** [comment]
Yep, i'm aware of this one. Something to do with modem initialization, so the first packet seems to get dropped. Unsure what has changed, if anything. The modem still decodes these transmissions fine, but we lose a byte of the first packet in the test recording so the CRC fails.

At some point I'll investigate what's going on.

**Comment by xssfox** [comment]
This doesn't seem to the be case anymore
